File: app/src/main/java/com/mytlogos/enterprise/background/api/model/ClientUpdateUser.java
package com.mytlogos.enterprise.background.api.model;

public class ClientUpdateUser {
    private final String name;
    private final String password;
    private final String newPassword;

    public ClientUpdateUser(String name, String password, String newPassword) {
        this.name = name;
        this.password = password;
        this.newPassword = newPassword;
    }

    public String getName() {
        return name;
    }

    public String getPassword() {
        return password;
    }

    public String getNewPassword() {
        return newPassword;
    }
}

File: app/src/main/java/com/mytlogos/enterprise/background/api/model/ClientUpdateUser.kt
package com.mytlogos.enterprise.background.api.model

class ClientUpdateUser(val uuid: String, val name: String, val password: String, val newPassword: String) {
    override fun equals(o: Any?): Boolean {
        if (this === o) return true
        if (o == null || javaClass != o.javaClass) return false
        val that = o as ClientUpdateUser
        if (uuid != that.uuid) return false
        if (name != that.name) return false
        if (password != that.password) return false
        return newPassword == that.newPassword
    }

    override fun hashCode(): Int {
        var result = uuid.hashCode()
        result = 31 * result + name.hashCode()
        result = 31 * result + password.hashCode()
        result = 31 * result + newPassword.hashCode()
        return result
    }
}